ReleaseProcessAutomation.UnitTests/SemanticVersioning/SemanticVersionTests.cs

```diff
@@ -173,6 +252,73 @@ public void GetCurrentPossibleVersionsHotfix_WithPreReleaseRC_ReturnsVersions ()
 
     Assert.That(possibleVersions, Is.EqualTo(expectedVersions));
   }
+  
+  [Test]
+  public void GetNextPossibleVersionsReleaseBranchFromDevelop_WithPreReleaseRC_ReturnsExpectedVersions ()
+  {
+    var parser = new SemanticVersionParser();
+    var expectedVersions = new SemanticVersion[]
+                           {
+                               parser.ParseVersion("1.3.5-rc.2"),
+                               parser.ParseVersion("1.3.5")
+                           };
+    var baseVersion = parser.ParseVersion("1.3.5-rc.1");
+
+    var possibleVersions = baseVersion.GetNextPossibleVersionsForReleaseBranchFromDevelop();
+
+    Assert.That(possibleVersions, Is.EqualTo(expectedVersions));
+  }
+  
+  [Test]
+  public void GetNextPossibleVersionsReleaseBranchFromDevelop_WithFullRelease_ReturnsExpectedVersions ()
```

Comment:
I think we also want a test for minor version, see GetNextPossibleVersionsDevelop_WithMinorVersion_ReturnsExpectedVersions and GetNextPossibleVersionsDevelop_WithMajorVersion_ReturnsExpectedVersions